sklearn/impute/tests/test_common.py

```diff
@@ -122,3 +122,43 @@ def test_imputers_pandas_na_integer_array_support(imputer, add_indicator):
     X_trans = imputer.fit_transform(X_df)
 
     assert_allclose(X_trans_expected, X_trans)
+
+
+# ConvergenceWarning will be raised by the IterativeImputer
+@pytest.mark.filterwarnings("ignore::sklearn.exceptions.ConvergenceWarning")
```

Comment:
Why is that? Could we get rid of the warning?

Reply:
It is the same filter warning used by the other test in the file. I changed the `tol` in `IteratorImputer` to avoid the warning for the file. I think this is okay, because the file is testing the imputer API and not the exact imputation method.